pipeline.sh

```diff
@@ -111,22 +104,25 @@ fi
 
 ##
 echo "### odgi bin"
-BIN=${GFA%.gfa}.w${w}.json
-BINPREF=${GFA%.gfa}_04_bin_w${w}
-SRTPREF=${GFA%.gfa}_03_bin
-/usr/bin/time -v -o ${SRTPREF}.time \
-ionice -c2 -n7 \
-$ODGI bin \
---json \
---idx=$SOG \
---bin-width=${w} \
-1> $BIN \
-2> ${BINPREF}.log
-
-if [ ! -f $BIN ]; then
+for w in 1 4 16 64 256 1000 4000 16000; do
```

Comment:
Is it possible to define the width array on where other variables are defined?
```bash
array=(1 4 16 64 256 1000 4000 16000)
```
That's because we use it later.